custom_components/mail_and_packages/helpers.py

```diff
@@ -33,10 +33,12 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.util import ssl
 from PIL import Image, ImageOps
+from voluptuous import Email, MultipleInvalid, Schema
 
 from .const import (
     AMAZON_DELIVERED,
     AMAZON_DELIVERED_SUBJECT,
+    AMAZON_DOMAINS,
```

Comment:
`AMAZON_DOMAINS` is no longer used, the user enters their amazon email domains in during configuration. That way we're not looping excessively.